packages/lsd-react/src/components/Calendar/Day.tsx

```diff
@@ -1,72 +1,78 @@
-import { useRef, useContext } from 'react'
-import { useDay } from '@datepicker-react/hooks'
-import { CalendarContext } from './Calendar.context'
+import { useCallback, useRef } from 'react'
+import { useCalendarContext } from './Calendar.context'
 import clsx from 'clsx'
-import { calendarClasses } from './Calendar.classes'
 import { Typography } from '../Typography'
+import {
+  getDayBorders,
+  isDateWithinRange,
+  isSameDay,
+  resetHours,
+} from '../../utils/date.utils'
+import { calendarClasses } from './Calendar.classes'
 
 export type DayProps = {
   day?: string
-  date: Date
+  index: number
+  fullMonthDays: Date[]
   disabled?: boolean
 }
 
-export const Day = ({ day, date, disabled = false }: DayProps) => {
+export const Day = ({
+  day,
+  index,
+  fullMonthDays,
+  disabled = false,
+}: DayProps) => {
+  const date = fullMonthDays[index]
+  const { mode, startDate, endDate, onDateSelect } = useCalendarContext()
   const dayRef = useRef(null)
-  const {
-    focusedDate,
-    isDateFocused,
-    isDateSelected,
-    isDateHovered,
-    isDateBlocked,
-    isFirstOrLastSelectedDate,
-    onDateSelect,
-    onDateFocus,
-    onDateHover,
-  } = useContext(CalendarContext)
+  const isToday = resetHours(date) === resetHours(new Date())
+  const isInDateRange =
+    mode === 'range' && isDateWithinRange(date, startDate, endDate)
+
+  const onClick = useCallback(() => onDateSelect(date), [date, onDateSelect])
 
-  const { onClick, onKeyDown, onMouseEnter, tabIndex } = useDay({
-    date,
-    focusedDate,
-    isDateFocused,
-    isDateSelected,
-    isDateHovered,
-    isDateBlocked,
-    isFirstOrLastSelectedDate,
-    onDateFocus,
-    onDateSelect,
-    onDateHover,
-    dayRef,
-  })
```

Comment:
One of the requests from @amirhouieh was:

> Is there a specific logic/routine implemented upon hover? the cursor seems shaking and unstable when hovering over the day numbers ?

I believe the culprit was this useDay hook - it adds some hovering logic [as we can see from its source code](https://github.com/t0gre/react-datepicker/blob/master/packages/hooks/src/useDay/useDay.ts#L32-L56). I removed this hook because as far as I can tell the hovering logic is not needed. If we do need it sometime in the future, it should be easy to implement it ourselves.